nextcloud-startup-script.sh

```diff
@@ -93,7 +93,7 @@ else
   apt install python-pip -y
  pip install \
      --upgrade pip \
-     apt-select
+     apt-select -m up-to-date -t 5 -c
```

Comment:
No, it fetches mirrors from a list provided by Ubuntu, But giving the user the option to choose at least minimizes the risk of getting a bad mirror. And with "bad" I mean some packages aren't found.